Packs/MISP/Integrations/MISPV3/MISPV3.py

```diff
@@ -1800,6 +1876,7 @@ def main():
     malicious_tag_ids = argToList(params.get('malicious_tag_ids'))
     suspicious_tag_ids = argToList(params.get('suspicious_tag_ids'))
     benign_tag_ids = argToList(params.get('benign_tag_ids'))
+    search_warninglists: bool = argToBoolean(params.get('search_warninglists'))
```

Comment:
please add here argToBoolean(params.get('search_warninglists', False)) - to have default value